app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com_d = command.split()
```

Comment:
It's better to assign each part of the command to separate variables, making them easier to understand when used.
```suggestion
    command, source_path, destination_path = command.split()
```